scripts/globals/effects/subtle_sorcery.lua

```diff
@@ -14,14 +14,12 @@ effect_object.onEffectGain = function(target, effect)
 end
 
 effect_object.onEffectTick = function(target, effect)
-    target:addMod(xi.mod.ENMITY, -20)
```

Comment:
The tick isn't set up when the effect is added and this would give the user -20 ENMITY per tick. According to https://www.bg-wiki.com/ffxi/Subtle_Sorcery:
> Reduces cumulative enmity from magic spells to 0.
Handling for the enmity would need to set CE directly to 0. I'm not sure if there's a global way that the effect could be applied to spells.

Reply:
This might be something that needs to be handled by `CMagicState::ApplyEnmity()`

Reply:
Set up `CMagicState::ApplyEnmity()` to check for Subtle Sorcery and set CE to 0 for all spells. In-game ability text states "Reduces the amount of enmity generated from magic spells" so the status check applies to any magic spell and is not restricted to elemental magic or enemy targeting only spells.